infra/helper.py

```diff
@@ -264,6 +273,38 @@ def coverage(run_args):
   pipe.communicate()
 
 
+def reproduce(run_args):
+  """Reproduces a testcase in the container."""
+  parser = argparse.ArgumentParser('helper.py reproduce')
+  parser.add_argument('project_name', help='name of the project')
+  parser.add_argument('fuzzer_name', help='name of the fuzzer')
+  parser.add_argument('testcase_path', help='absolute path of local testcase')
```

Comment:
Looks like docker does the path translation automatically.

docker run --rm -i --cap-add SYS_PTRACE -v /usr/local/google/home/aarya/build/oss-fuzz/build/out/libarchive:/out -v /usr/local/google/home/aarya/Downloads/fuzz-3-libarchive_fuzzer:/testcase -t ossfuzz/base-runner reproduce /out/libarchive_fuzzer